### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by MPvHarmelen**

### CODE REVIEW

**Comment by martijnarts** [comment]
Look who's back!

File: board.py [hunk 1]
```diff
@@ -260,12 +261,17 @@ class AIBoard(Board):
...
         logger.debug('args: {}'.format(args))
         logger.debug('kwargs: {}'.format(kwargs))
+        self.mutations = {}
         super(AIBoard, self).__init__(*args, **kwargs)
         self.mutations = {piece: [] for piece in self.pieces if piece.is_AI()}
+        self.reset()
+
+    def reset(self):
+        """Reset the board"""
+        super(AIBoard, self).reset()
```
Comment (martijnarts) [inline board.py:271]: How very Python 2.

Comment (MPvHarmelen) [reply board.py:271]: What is the Python 3 way?

Comment (martijnarts) [reply board.py:271]: `super().reset()`

**Pull request merged**